phoenix-core/src/it/java/org/apache/phoenix/end2end/index/IndexMetadataIT.java

```diff
@@ -100,7 +100,7 @@ private static void assertActiveIndex(Connection conn, String schemaName, String
         String fullTableName = SchemaUtil.getTableName(schemaName, tableName);
         conn.createStatement().executeQuery("SELECT count(*) FROM " + fullTableName).next(); // client side cache will update
         PhoenixConnection pconn = conn.unwrap(PhoenixConnection.class);
-        pconn.getMetaDataCache().getTable(new PTableKey(pconn.getTenantId(), fullTableName)).getIndexMaintainers(ptr, pconn);
```

Comment:
Probably good to factor out this method change (i.e. having PhoenixConnection.getTable()) into a separate pull to master as it appears to have generated a lot of diffs unrelated to transactions.